next.config.js

```diff
@@ -4,7 +4,7 @@ const nextConfig = {
   swcMinify: true,
   compiler: { styledComponents: true },
   eslint: {
-    dirs: ["components", "constants", "hooks", "pages", "queries", "stories", "types"],
+    dirs: ["src"],
```

Comment:
this is one of the most insane things to me about the whole idea of not using an src directory (which libs like nextjs actually do by default) — you have to manually specify the folders that eslint looks at in the config. so if you add a new one and forget to add it here, eslint just ignores it. totally ridiculous